sample-queries/sample-queries.json

```diff
@@ -1241,7 +1241,7 @@
     },
     {
         "id": "3f05c89a-cd21-4570-85ac-cb67c10d63d4",
-        "category": "Microsoft Teams (beta)",
+        "category": "Users",
```

Comment:
You can move this category up to where the other **Users** category sample queries are.